index.js

```diff
@@ -31,7 +31,20 @@
     }
   }
 
+  function goatifyAll (obj) {
+    for (var k in obj) {
+      if (obj.hasOwnProperty(k)) {
+        var v = obj[k]
+        if (typeof v === 'function') { // FIXME IIRC this fails on old browsers
```

Comment:
IIRC this would fail on previous major version of phantomjs (don't know what JavaScriptCore version it used though)